### ISSUE TITLE
[JOSS] Paper

### ISSUE THREAD

**Issue by RMeli** [Degiacomi-Lab/molearn#7]
# Statement of need

I think the first paragraph could do with a few more relevant citations? For example the following sentences?

* Thus, their function emerges from characteristic conformational dynamics. 
* Progress in these areas has been spearheaded by the development of a diverse palette of dedicated experimental techniques.

Is GNNs a common acronym for Generative Neural Networks? GNN is commonly used for Graph Neural Networks too, so it might cause some confusion. (Even if it is defined in the text, I kept reading it in my head as Graph Neural Networks.)

Suggested changes:

```diff
-The issue is that developing an Machine Learning model to study biomolecular dynamics is a lengthy process.
+The issue is that developing a Machine Learning model to study biomolecular dynamics is a lengthy process.
```

```diff
-Here we present molearn, a Python framework facilitating the implementation of generative neural networks learning protein conformational spaces from examples obtained via experiments or MD simulations.
+Here we present molearn, a Python framework facilitating the implementation of GNNs learning protein conformational spaces from examples obtained via experiments or MD simulations.
```

# Usage

Suggested change:

```diff
-Results obtainable via `molearn` are exemplified in [@Ramaswamy2021].
+Results obtainable via `molearn` are exemplified in @Ramaswamy2021.
```

# Citations

Should citations for the following dependencies (listed in the `README.md`) be added to the paper?

* NumPy
```
@article{harris2020array,
  title={Array programming with NumPy},
  author={Harris, Charles R and Millman, K Jarrod and Van Der Walt, St{\'e}fan J and Gommers, Ralf and Virtanen, Pauli and Cournapeau, David and Wieser, Eric and Taylor, Julian and Berg, Sebastian and Smith, Nathaniel J and others},
  journal={Nature},
  volume={585},
  number={7825},
  pages={357--362},
  year={2020},
  publisher={Nature Publishing Group UK London}
}
```
* geomloss
```
@inproceedings{feydy2019interpolating,
    title={Interpolating between Optimal Transport and MMD using Sinkhorn Divergences},
    author={Feydy, Jean and S{\'e}journ{\'e}, Thibault and Vialard, Fran{\c{c}}ois-Xavier and Amari, Shun-ichi and Trouve, Alain and Peyr{\'e}, Gabriel},
    booktitle={The 22nd International Conference on Artificial Intelligence and Statistics},
    pages={2681--2690},
    year={2019}
}
```
* biobox
```
@article{rudden2022biobox,
  title={Biobox: a toolbox for biomolecular modelling},
  author={Rudden, Lucas SP and Musson, Samuel C and Benesch, Justin LP and Degiacomi, Matteo T},
  journal={Bioinformatics},
  volume={38},
  number={4},
  pages={1149},
  year={2022},
  publisher={Oxford University Press}
}
```
* modeller
* cctbox
* MDAnalysis
```
@article{michaud2011mdanalysis,
  title={MDAnalysis: a toolkit for the analysis of molecular dynamics simulations},
  author={Michaud-Agrawal, Naveen and Denning, Elizabeth J and Woolf, Thomas B and Beckstein, Oliver},
  journal={Journal of computational chemistry},
  volume={32},
  number={10},
  pages={2319--2327},
  year={2011},
  publisher={Wiley Online Library}
}

@inproceedings{gowers2016mdanalysis,
  title={MDAnalysis: a Python package for the rapid analysis of molecular dynamics simulations},
  author={Gowers, Richard J and Linke, Max and Barnoud, Jonathan and Reddy, Tyler JE and Melo, Manuel N and Seyler, Sean L and Domanski, Jan and Dotson, David L and Buchoux, S{\'e}bastien and Kenney, Ian M and others},
  booktitle={Proceedings of the 15th python in science conference},
  volume={98},
  pages={105},
  year={2016},
  organization={SciPy Austin, TX}
}
```

(On this note, is NGLview missing from the list of GUI dependencies in the `README.md`?)

---

https://github.com/openjournals/joss-reviews/issues/5523

**Comment by degiacom** [comment]
Thank you for your feedback, here is a summary of the changes made to the paper and `README.md`:

* We agree that the GNN acronym can be confusing, so we replaced it with everywhere in the text.
* We applied the modifications to the text as suggested
* In the Introduction, we added a reference to the 2019 Annual Reviews opening article by Chris Dobson, discussing experimental techniques developed to study protein structure, as well as the importance of protein dynamics in determining their function.
* Concerning the dependencies described in the `README.md` file, we have now added references to Biobox and MDAnalysis (used for I/O) and numpy (our analysis part of the package returns arrays). geomloss is associated with an experimental feature we only recently started investigating, we would rather not highlight this feature at this stage.
* When discussing "Model Analysis" (lines 58-61), we realised the we used the word "model" referring to both the neural network, and the protein structures it generates. We have rephrased two sentences to disambiguate these two concepts.
* We now mention NGLView in the `README.md` file, and link to its webpage.
